_config/config.yml

```diff
@@ -5,6 +5,12 @@ Page:
   extensions:
     - DNADesign\Elemental\Extensions\ElementalPageExtension
 ---
+Name: nswdpc-content-leftandmain
+---
+SilverStripe\Admin\LeftAndMain:
+  extra_requirements_css:
+    - 'vendor/nswdpc/silverstripe-content-boilerplate/client/static/css/leftandmain.css'
```

Comment:
I'd recommend using the module loader reference here instead of the full path
`- 'nswdpc/silverstripe-content-boilerplate:client/static/css/leftandmain.css'`

Reply:
Well spotted, thanks Michael!